test/MainEvent.cxx

```diff
@@ -131,9 +131,12 @@ int main(int argc, char **argv)
    Int_t branchStyle = 1; //new style by default
    if (split < 0) {branchStyle = 0; split = -1-split;}
 
+   ROOT::EnableImplicitMT(4);
```

Comment:
As a command line flag so that the default stay as is *and* one can test with and without it with the same build.